### ISSUE TITLE
Button to stop animations

### ISSUE THREAD

**Issue by craigfrancis** [craigfrancis/stop-animations#1]
"You need to press escape to stop he animations. This means you need to ensure your other apps don't use escape. It would be nice to have a button for cases where you cannot make the other apps ignore escape."

From Tim Josling

**Comment by craigfrancis** [comment]
I'll see about adding a button to the toolbar, one that can be hidden... and maybe an option to allow the keyboard shortcut to be changed.

**Comment by Klortho** [comment]
Thanks for the effort, but the first site I tried to use it on captures the <esc> key, so this feature would be useful.

**Comment by craigfrancis** [comment]
Can you tell me which site is doing this?

As I'd like to run some tests when I finally get a chance to look at this (unfortunately other work is getting in the way at the moment).

**Comment by Klortho** [comment]
Sure, https://www.deepdotweb.com/. I'm not 100% sure that it's grabbing the `esc` key, but the plugin didn't work, and I figure that's the most likely culprit.

**Comment by craigfrancis** [comment]
I've finally managed to add support for custom shortcuts... and I've some other changes that Chrome is requiring all extension authors to make.

If you go to "Extensions", make sure the Stop Animation extension is version 2.0 (or later)... and then use the 3 line menu in the top left, and choose "Keyboard shortcuts" (no idea why they have hidden it like that).

I looked at adding a toolbar button, but it gets a bit complicated trying to support how most people use the extension (with just a simple keyboard shortcut, and nothing wasting space in their toolbar) vs having something that's always visible to toggle the state... I might look at this again in the future, but I've run out of time, and need to get back to normal work.

As to the noted website, that seems to be offline now, so I can't check if that issue has been fixed.
